File: java-hangman-app/src/test/java/ru/polina_project/hangman/util/MaskerTest.java
package ru.polina_project.hangman.util;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class MaskerTest {

    @Test
    void doMask() {
        // test input
        String testWordToMask = "polina";
        char testSymbolToMask = '*';

        // test result
        String expectedResult = "******";

        assertEquals(Masker.doMask(testSymbolToMask, testWordToMask), expectedResult);
    }
}

File: kotlin-hangman-app/src/test/kotlin/ru/polina_project/hangman/util/MaskerTest.kt
package ru.polina_project.hangman.util

import org.junit.jupiter.api.Assertions.assertEquals
import kotlin.test.Test

class MaskerTest {
@Test
fun testDoMask() {
    val maskSymbol = '*'
    val wordToMask = "hangman"

    val expectedMaskedWord = "*******"

    val actualMaskedWord = doMask(maskSymbol, wordToMask)
    assertEquals(expectedMaskedWord, actualMaskedWord)
}
}